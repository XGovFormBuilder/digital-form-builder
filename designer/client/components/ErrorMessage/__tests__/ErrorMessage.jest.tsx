import React from "react";
import { render, cleanup } from "@testing-library/react";
import { ErrorMessage } from "..";

describe("ErrorMessage component", () => {
  afterEach(cleanup);

  it("matches snapshot", async () => {
    const { debug, asFragment } = render(
      <ErrorMessage className="123">Error 123</ErrorMessage>
    );
    debug();
    expect(asFragment()).toMatchSnapshot();
  });
});
