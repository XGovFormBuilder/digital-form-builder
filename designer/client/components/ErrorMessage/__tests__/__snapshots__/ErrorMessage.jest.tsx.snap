// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage component matches snapshot 1`] = `
<DocumentFragment>
  <span
    class="govuk-error-message 123"
  >
    <span
      class="govuk-visually-hidden"
    />
     Error 123
  </span>
</DocumentFragment>
`;
